deploy/osd-cluster-admin/06-osd-impersonators.Group.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: osd-impersonators
```

Comment:
Do we want to prefix this group, if we want the customer to be able to edit it? Will this group's membership be erased by the SelectorSyncSet?

Reply:
No, the group membership will not be reset automatically. I'm not sure that we _want_ the customers to edit it, but with impersonation they will have access. The specifics on how the workflow will interface with OCM hasn't been fully realized so I'm hesitant to default to an unprotected namespace.